apps/webstore/src/app/pages/auth/unsubscribe/unsubscribe.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="full-page">
+    <span
+        *ngIf="isLoading"
+        class="spinner-border spinner-border-sm"
+        style="margin: 5px"
+        role="status"
+        aria-hidden="true"
+    ></span>
+    <div *ngIf="!isLoading && unsubscribeSuccessful">
+        <h3 i18n="@@unsubscribe-success">Unsubscribe success</h3>
+        <p i18n="@@have-a-good-day">
+            Thank you for your time. Have a good day.
+        </p>
+    </div>
+    <div *ngIf="!isLoading && !unsubscribeSuccessful" class="failed">
+        <h3 i18n="@@unsubscribe-failed">Unsubscribe failed</h3>
```

Comment:
Should specify the reason for failure, based on the API response
So invalid token, never subscribed, etc
Don't just parse straight from the response, instead read the status and adjust the alert accordingly only for specific cases (don't say "500 internal Server Error" etc)

You can find an example in the verification pages